spec/requests/users_spec.rb

```diff
@@ -1,6 +1,37 @@
 require 'swagger_helper'
 
 RSpec.describe 'users', type: :request do
+  path '/users' do
+    parameter name: 'page', in: :query, type: :integer, default: 1, required: false
```

Comment:
It is better to write your query parameters inside get block to avoid potentials bugs if someone decide to add other requests under path '/users'

Reply:
Fixed